src/url-params.ts

```diff
@@ -163,10 +163,20 @@ export class AnalyticsUrlParams {
 	/**
 	 * @return full query parameter string that can be appended to URLs
 	 */
-	getQueryString(): string {
-		return [this.getDeviceIdsQueryString(), this.getSessionIdQueryString()]
-			.filter(x => x)
-			.join('&');
+	getQueryString(destinationHostname?: string): string {
+		const regex = /([a-zA-Z0-9-]+)(\.[a-zA-Z]{2,5})?(\.[a-zA-Z]+$)/g;
+		const actualDomain = window.location.hostname.match(regex);
```

Comment:
Shall we support `currentUrl` as parameter as well? Is it safe to assume that `UrlParams` class always has access to `window`?

I feel like there could be a case that a component wants to ask for URL params for other places beyond `window.location`.

Reply:
Just in case this happens I added the currentURL parameter to the function and the logic to parse it in case there is one, otherwise it tries to use window.location.hostname